tests/Integration/ApiPlatform/DomainSerializerTest.php

```diff
@@ -60,7 +60,7 @@ public function testDenormalize($dataToDenormalize, $denormalizedObject, ?array
     public function testDenormalizeWithEmptyValues(): void
     {
         $serializer = self::getContainer()->get(DomainSerializer::class);
-        $value = $serializer->denormalize(null, 'PrestaShopBundle\ApiPlatform\Resources\Product', null, []);
+        $value = $serializer->denormalize(null, 'PrestaShop\Module\APIResources\ApiPlatform\Resources\Product', null, []);
```

Comment:
```suggestion
        $value = $serializer->denormalize(null, Product::class, null, []);
```
Just a suggestion, not blocking

Reply:
To me, its preferable to use the `Product::class` way on this case.

Reply:
btw this file was just copied back from the module I did not change it in any way :/